### PR TITLE
add disinfect plugin

### PR DESCRIPTION
**Pull request by genediazjr**

### CODE REVIEW

File: lib/plugins.js [hunk 1]
```diff
@@ -198,6 +198,10 @@ exports.categories = {
         }
     },
     'Utility': {
+        'disinfect': {
+            url: 'https://github.com/genediazjr/disinfect',
+            description: 'Request query, payload, and params sanitation'
```
Comment (Marsup) [inline lib/plugins.js]: Meant sanitization here ?

Comment (devinivy) [inline lib/plugins.js]: I think sanitation and sanitization could both work here.

Comment (Marsup) [inline lib/plugins.js]: OK, thought it was a typo.

Comment (mtharrison) [inline lib/plugins.js]: Really? I've never seen "sanitation" used in software terms before.

Comment (genediazjr) [inline lib/plugins.js]: I agree, I think it would be proper to use sanitization. I'll provide an update.

**Pull request merged**